src/pages/Admin/Charity/EditProfile/schema.ts

```diff
@@ -16,6 +16,7 @@ export const VALID_MIME_TYPES: ImageMIMEType[] = [
 ];
 
 export const MAX_SIZE_IN_BYTES = 1e6;
+export const MAX_CHARS = 4000;
```

Comment:
bug: 
<img width="925" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/a2075ede-d8f3-401d-810a-ca0f1189ce3e">

REPRO:
put more than `4000` chars - error appears
delete to less than `4000` - error should disappear

Reply:
This is a bug on `master`, so unrelated to this PRs focus. @SovereignAndrey do we have a ticket for this?

Reply:
This PR introduces new schema structure for Richtext, no? Would it make sense to expect that those schema handle error (whether previously existing) as they arise and resolved ?

Reply:
you applied explicit validation via schema `richTextContent` to `profile.overview`. Do we really need another PR just to say that previous  `RichTextEditor.onError` is not needed anymore? - can't we just delete that in this PR ?

Reply:
created PR to do just this
* https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2751

Reply:
@SovereignAndrey @ap-justin 3 reasons I like to keep my PRs focused:
1. Easier to track in Linear what was done, when and by whom
2. Easier to review
3. Easier to revert if something goes wrong